src/app/Instance/InstancePage/InstancePage.tsx

```diff
@@ -272,16 +264,40 @@ const InstancePage = (): JSX.Element => {
                   <TabTitleText>{t("common.errorHandling")}</TabTitleText>
                 )
               }
-            >
-              <PageSection>
-                <ErrorHandlingTabContent
-                  bridge={bridge}
-                  isBridgeLoading={isBridgeLoading}
-                />
-              </PageSection>
-            </Tab>
+            />
           </Tabs>
         </PageSection>
+        <PageSection isFilled padding={{ default: "noPadding" }}>
+          <TabContent
```

Comment:
Switching to the separate content tabs setup, where tabs content is not nested inside tab links ([docs example](https://www.patternfly.org/v4/components/tabs#separate-content)).
I needed this to explicitly use `TabContent` and modify it a bit.